### ISSUE TITLE
[Docs] Flat Config with TypeScript

### ISSUE THREAD

**Issue by raky291** [import-js/eslint-plugin-import#3112]
It would be very helpful if we have more documentation related to Flat Config with TypeScript?

Do we still need to configure "import/resolver" with Flat Config? Do we have any example?

**Comment by ljharb** [comment]
Yes, all the config requirements are identical - flat config is just a different way to express an eslintrc config.
